### ISSUE TITLE
Add dialogue after clicking on eye-symbol to follow object

### ISSUE THREAD

**Issue by daria-sl** [globe-swiss/phaenonet-client#192]
According to Dani, it would be in our interest to insert a dialogue after having clicked the eye-symbol to follow object in the public object view. Like this, we could make it possible for users to follow all objects of the object owner, if wished for. 
![public_object2](https://user-images.githubusercontent.com/65400300/134862443-9773d6b4-1fb4-40f4-97aa-402d266fb059.png)

@pgoellnitz what do you think of this? To me it seems a good idea (we want to create more activities, in general) - it would also make sense only to have this dialogue in the public object view (and not when a user is followed in his/her public profile), because in the latter case, we do not want to offer the possibility of "downscaling" only to a single object.

- [ ] @daria-sl define texts in all three languages

**Comment by daria-sl** [comment]
@daria-sl if change is done, adaptions in PhaenoNet instructions for teachers are necessary.

**Comment by pgoellnitz** [comment]
Hi @daria-sl, didn't we add this to the scope of #147? Anyway we should tackle this after the redesign is finished as the screenshot already has the new design.

**Comment by daria-sl** [comment]
Yes @pgoellnitz let‘s tackle this after the redesign 👍🏼 It is not part of #147 .
